common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -303,6 +313,24 @@ def setUp(self):
         )
         self.thread_page_1.visit()
 
+        self.thread_page_1.a11y_audit.config.set_rules({
```

Comment:
Sometimes these `set_rules` calls are in the `test_page_accessibility` calls with the calls to `check_for_accessibility_errors`, and sometimes they are in the setup call.  I'm not sure which is better.  Possibly in the `test_page_accessibility` call?  Could you propose one vs the other, and them make these all consistent?